src/apps/immich/readme.md

```diff
@@ -0,0 +1,165 @@
+---
+# author: giga
+category: [Privacy]
+icon: device-camera-video 
+---
+
+## Prerequisites
+
+[!ref icon="check-circle" text="Complete First Steps"](/first-steps/1-vps-setup.md)
+
+## What is Immich
+
+Immich is a self-hosted, open-source platform for managing and sharing your personal media library. It provides a simple and intuitive interface for organizing and streaming your movies, TV shows, and music.
```

Comment:
As far as I know, Immich serves as a replacement for Google Photos and similar other clients, and has nothing to do with streaming movies, tv shows or music.